main_pipeline/tests/util_tests.py

```diff
@@ -0,0 +1,107 @@
+import unittest
```

Comment:
This and `main_pipeline/tests/test_utils/__init__.py` should be deleted as well.